index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please restore the title

Reply:
* Remember to explain why you request a change - it might be obvious for you but not for someone else.